app/static/css/index.css

```diff
@@ -1,20 +1,39 @@
+#login h2 {
+    margin-bottom: 10px;
+}
 #login input {
-  width:250px;
+    display: block;
+    width: 100%;
+    padding: .375rem .75rem;
+    font-size: 1rem;
+    line-height: 1.5;
+    color: #495057;
+    background-color: #fff;
+    background-clip: padding-box;
+    border: 1px solid #ced4da;
+    border-radius: .25rem;
+    transition: border-color .15s ease-in-out,box-shadow .15s ease-in-out;
+}
+#login input:focus {
+    outline: none;
+    border-color: red;

```

Comment:
Why red? Makes me feel like there's an error.

Reply:
The original application before the changes also had a red border when a text box was selected, so I thought it was part of the style. Turns out it only turns red when the text box is empty. Nice catch, will fix.